app/helpers/invoices_helper.rb

```diff
@@ -6,4 +6,12 @@ def get_items
   def get_clients
     current_user.company.clients.map{ |client|[client.name,client.id]}
   end
-end
+
+  def get_address(invoice)
+    if invoice.address.present?
+      "#{invoice.address.street_1} #{invoice.address.street_2}, <br/>#{invoice.address.city}, #{invoice.address.state}, <br/>#{invoice.address.pincode.to_s}".html_safe
```

Comment:
Use `\n` instead of hardcoding html tag in string `<br/>` and call [simple_format](http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-simple_format)